docs/user_guides/fs/transformation_functions.md

```diff
@@ -0,0 +1,163 @@
+
```

Comment:
Add a warning to tell that definition transformation function within a Jupyter notebook is only supported in Python Kernel. In a PySpark Kernel transformation function have to defined as modules and imported.